gen3/bin/kube-dev-namespace.sh

```diff
@@ -17,10 +17,10 @@ if [[ -z "$GEN3_NOPROXY" ]]; then
   export https_proxy=${https_proxy:-'http://cloud-proxy.internal.io:3128'}
   export no_proxy=${no_proxy:-'localhost,127.0.0.1,169.254.169.254,.internal.io,logs.us-east-1.amazonaws.com,kibana.planx-pla.net'}
 fi
-
+#the first argument need to be <namespace> string
 namespace="$1"
 if ! shift || [[ -z "$namespace" || (! "$namespace" =~ ^[a-z][a-z0-9-]*$) || "$namespace" == "$vpc_name" ]]; then
-  gen3_log_err "Use: bash kube-dev-namespace.sh namespace, namespace is alphanumeric"
+  gen3_log_err "new namespace has to be different than vpc name"
```

Comment:
Instead of replacing the error message can you combine them? Something like 
```
Use: bash kube-dev-namespace.sh namespace, namespace is alphanumeric and new namespace has to be different than vpc name
```